source/extensions/transport_sockets/tls/context_config_impl.h

```diff
@@ -99,6 +96,11 @@ class ContextConfigImpl : public virtual Ssl::ContextConfig {
 
   Ssl::HandshakerFactoryCb handshaker_factory_cb_;
   Ssl::HandshakerCapabilities capabilities_;
+
+protected:
+  std::vector<Secret::TlsCertificateConfigProviderSharedPtr> tls_certificate_providers_;
+  Secret::CertificateValidationContextConfigProviderSharedPtr
+      certificate_validation_context_provider_;
```

Comment:
just double-checking, is the re-order fine? also style guide says in a class it should be `public` -> `protected` -> `private` order.

Reply:
done